armsrc/appmain.c

```diff
@@ -2663,7 +2662,11 @@ static void PacketReceived(PacketCommandNG *packet) {
             break;
         }
         case CMD_STATUS: {
-            SendStatus();
+            Dbprintf("Packet Len:%d", packet->length);
```

Comment:
remove debug statement

Reply:
Oh I forgot to delete this. Thanks for remind me.